WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -61,16 +61,18 @@ private set
 		public Func<Uri> BaseUriGetter { get; }
 		public EndPoint? TorSocks5EndPoint { get; private set; }
 
-		/// <inheritdoc/>
+		/// <summary>
+		/// Whether each HTTP(s) request should use a separate Tor circuit by default or not to increase privacy.
+		/// </summary>
 		public bool DefaultIsolateStream { get; }
```

Comment:
Consider changing this maybe this is just `IsolateStream` not default anymore. There is no method where you can choose to isolate or not - so default does not make sense.